modules/ROOT/pages/tls-configuration.adoc

```diff
@@ -49,7 +49,13 @@ A well-known Certificate Authority (CA) can generate certificates, or you can ge
 
 == Keystores and Truststores
 
-The `tls:trust-store` and `tls:key-store` elements in a Mule configuration can reference a specific certificate and key, but if you don't provide values for `tls:trust-store`, Mule uses the default Java truststore. Java updates the default trust store when you update Java, so getting regular updates is recommended to keep well-known CA certificates up-to-date.
+In a Mule configuration, the `tls:trust-store` and `tls:key-store` elements allow you to reference specific certificates and keys. If you don't specify a `tls:trust-store`, Mule uses the default Java truststore, which is maintained by Java and is updated automatically when Java is updated. This helps ensure that certificates from Certificate Authorities (CAs) remain current.
+
+Using a custom Java Trust Store is not the standard approach in MuleSoft and can lead to connectivity issues with the MuleSoft control plane when MuleSoft certificates are renewed. 
+
+[IMPORTANT]
+We recommend using the default Java truststore to prevent connectivity disruptions. If you use a custom truststore, you are responsible for managing and updating it.
```

Comment:
```suggestion
[IMPORTANT]
We recommend using the default Java truststore to prevent connectivity disruptions. Using a custom Java truststore isn’t the standard approach in MuleSoft and can cause connectivity issues with the MuleSoft control plane when MuleSoft certificates are renewed. If you choose to use a custom truststore, you’re responsible for managing and updating it.
```
Another shorter option: 

[IMPORTANT]
Use the default Java truststore to avoid connectivity issues. If you use a custom truststore, you’ll need to manage and update it when MuleSoft certificates are renewed.